src/main/kotlin/me/partlysanestudios/partlysaneskies/events/skyblock/dungeons/DungeonStartEvent.kt

```diff
@@ -0,0 +1,21 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+
+package me.partlysanestudios.partlysaneskies.events.skyblock.dungeons
+
+import kotlin.reflect.KFunction
+
+class DungeonStartEvent {
```

Comment:
Wait also please use this event to reset the required secret check (only at the start, ignore the finish)